suite-native/module-settings/src/components/ProductionDebug.tsx

```diff
@@ -0,0 +1,32 @@
+import React, { ReactNode } from 'react';
+import { Pressable } from 'react-native';
+
+import { useAtom } from 'jotai';
+
+import { isDevelopOrDebugEnv } from '@suite-native/config';
+import { atomWithUnecryptedStorage } from '@suite-native/storage';
+
+type ProductionDebugProps = {
+    children: ReactNode;
+};
+
+let tapsCount = 0;
+export const isDevButtonVisibleAtom = atomWithUnecryptedStorage<boolean>(
+    'isDevUtilsScreenVisible',
+    isDevelopOrDebugEnv(),
+);
+
+export const ProductionDebug = ({ children }: ProductionDebugProps) => {
+    const [_, setIsDevButtonVisible] = useAtom(isDevButtonVisibleAtom);
```

Comment:
you can use `useSetAtom` if you don't need it's value, and it will also not rerender component with every change

Reply:
Fixed